src/core/Akka.Remote.TestKit.Tests/RemoteConnectionSpecs.cs

```diff
@@ -62,13 +69,13 @@ public RemoteConnectionSpecs() : base(Config)
                 var address = RARP.For(Sys).Provider.DefaultAddress;
 
                 // have the client send a message to the server
-                await client.WriteAndFlushAsync(new Hello("test", address));
+                client.WriteAndFlushAsync(new Hello("test", address));
                 var hello = serverProbe.ExpectMsg<Hello>();
```

Comment:
Might be noteworthy that by removing the above await. This ExpectMsg has by default 3s to receive the Hello message.